algosdk/util.py

```diff
@@ -15,7 +16,7 @@ def microalgos_to_algos(microalgos):
     Returns:
         int or float: how many algos
     """
-    return microalgos/constants.microalgos_to_algos_ratio
+    return decimal.Decimal(microalgos)/constants.microalgos_to_algos_ratio
```

Comment:
We use `decimal` here to make sure we properly get the decimal amount of Algos?

Reply:
Yes, because python's built in floating point arithmetic doesn't have enough significant digits when you get into higher numbers.